lnbits/static/i18n/en.js

```diff
@@ -33,6 +33,8 @@ window.localisation.en = {
   reset_defaults_tooltip: 'Delete all settings and reset to defaults.',
   download_backup: 'Download database backup',
   name_your_wallet: 'Name your %{name} wallet',
+  wallet_topup_ok:
+    'Success creating virtual funds (%{amount} sats). Payments depend on actual funds on funding source.',
```

Comment:
shouldnt this say "(%{credit} sats)" in each translation ?

Reply:
no, there is an object which is passed as param:

```js
            message: this.$t('wallet_topup_ok', {
              amount: credit
            }),
```

Reply:
no, amount is fine, if you look into the `this. $q.notify()`